corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -136,25 +123,26 @@ def handle(self, domain, months, csv, **options):
             .domain(domain).values_list("_id", flat=True)
         )
 
-        self._doc_counts()
-        self._forms_per_user_per_month()
-        self._cases_created_per_user_per_month()
-        self._cases_updated_per_user_per_month()
-        self._case_transactions()
-        self._case_indices()
-        self._synclogs()
-        self._case_to_case_index_ratio()
-        self._ledgers_per_case()
-        self._attachment_sizes()
-        self._ucr()
-
-        self._output_stats()
-
-    def _doc_counts(self):
-        ResourceModel.set_stat(ResourceModel.FORMS_TOTAL, FormES().domain(self.domain).count())
-        ResourceModel.set_stat(ResourceModel.CASES_TOTAL, CaseES().domain(self.domain).count())
-
-    def _forms_per_user_per_month(self):
+        self.collect_doc_counts()
+        self.collect_forms_per_user_per_month()
+        self.collect_cases_created_per_user_per_month()
+        self.collect_cases_updated_per_user_per_month()
+        self.collect_case_transactions()
+        self.collect_case_indices()
+        self.collect_synclogs()
+        self.collect_devicelogs()
+        self.collect_case_to_case_index_ratio()
+        self.collect_ledgers_per_case()
+        self.collect_attachment_sizes()
+        self.collect_ucr_data()
+
+        self.output_stats()
+
+    def collect_doc_counts(self):
+        resource_model.__setitem__('forms_total', FormES().domain(self.domain).count())
```

Comment:
I think this would now be
```py
resource_model['forms_total'] = FormES().domain(self.domain).count()
...
```
(and the same for everywhere else)

Reply:
(omw, Charl... :facepalm: )